packages/elements/src/datetime-picker/__test__/datetime-picker.default.test.js

```diff
@@ -3,25 +3,48 @@ import { fixture, expect, elementUpdated } from '@refinitiv-ui/test-helpers';
 // import element and theme
 import '@refinitiv-ui/elements/datetime-picker';
 import '@refinitiv-ui/elemental-theme/light/ef-datetime-picker';
-
-const INPUT_FORMAT = {
-  DATE: 'dd-MMM-yyyy',
-  DATETIME: 'dd-MMM-yyyy HH:mm',
-  DATETIME_AM_PM: 'dd-MMM-yyyy hh:mm aaa',
-  DATETIME_SECONDS: 'dd-MMM-yyyy HH:mm:ss',
-  DATETIME_SECONDS_AM_PM: 'dd-MMM-yyyy hh:mm:ss aaa'
-};
+import { inputElement, inputToElement } from './utils';
 
 describe('datetime-picker/DatetimePicker', () => {
+  describe('DOM Structure', () => {
+    it('DOM structure is correct', async () => {
+      const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04"></ef-datetime-picker>');
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+    it('DOM structure is correct when opened', async () => {
+      const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04" opened></ef-datetime-picker>');
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+    it('DOM structure is correct when range', async () => {
+      const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04" range opened></ef-datetime-picker>');
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+    it('DOM structure is correct when duplex', async () => {
+      const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04" duplex opened></ef-datetime-picker>');
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+    it('DOM structure is correct when timepicker', async () => {
+      const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04" timepicker opened></ef-datetime-picker>');
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+    it('DOM structure is correct when timepicker and with-seconds', async () => {
+      const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04" timepicker with-seconds opened></ef-datetime-picker>');
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+    it('DOM structure is correct when range timepicker', async () => {
+      const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04" timepicker range opened></ef-datetime-picker>');
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+  });
```

Comment:
Add here a test case with formatOptions as well, to test date-only; time-only and date-time test scenarios.

Reply:
Added.